Makefile

```
@@ -1,8 +1,8 @@
 PREFIX ?= /usr/local
 
 install: bin/n
-	mkdir -p $(PREFIX)/$(dir $<)
-	cp $< $(PREFIX)/$<
+	mkdir -p $(PREFIX)/$(dir bin/n)
```

Simplify the (now) hard-coded directory name:
```suggestion
	mkdir -p $(PREFIX)/bin
```